tests/network/setup_multimachine.pm

```diff
@@ -41,6 +41,12 @@ sub run {
             disable_and_stop_service('NetworkManager', ignore_failure => 1);
             assert_script_run 'systemctl start  wicked';
         }
+        #if server runing on SLED
```

Comment:
Why are you switching from `NetworkManager` to `wicked` ?
`NetworkManager` is default on SLED and any test should run with it.

Reply:
ok, I change to configure SLED NetwokrManager . Thanks